### DISCUSSION TITLE
How to refine personal feed?

### DISCUSSION THREAD

**Discussion by daskol** [community/community#13172] [Other Feature Feedback, Questions, & Ideas] [answered]
I noticed that there is only option to block (or report) a user in a feed. Obviously, some implicit feedback is used like one's own stars and follower/-ing stats. Are there is meaningful ways to effect recsys for better recommendations?

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
